index.html

```diff
@@ -1,106 +1,58 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Super Motivation</title>
-    <style>
-      .card {
-        width: 550px;
-        height: 377px;
-        background-color: rgb(0, 225, 255);
-        border-radius: 16px;
-        display: flex;
-        flex-direction: row;
-      }
-      .image {
-        background-color: rgb(0, 7, 8);
-        width: 50%;
-      }
-      .image .vergil {
-        width: 250px;
-        height: 250px;
-        margin: 12px;
-      }
-      .image .demotivation {
-        font-size: 32px;
-        color: rgb(255, 255, 255);
-        font-family: serif;
-        font-weight: bold;
-        margin: 16px 0;
-        line-height: 1.1;
-        margin-left: 64px;
-      }
-      .content {
-        background-color: rgb(190, 225, 255);
-        width: 50%;
-        padding: 24px;
-        display: flex;
-        flex-direction: column;
-        justify-content: space-between;
-      }
-      .content .type {
-        font-size: 12px;
-        color: grey;
-        font-family: sans-serif;
-      }
-      .content .title {
-        font-size: 32px;
-        color: black;
-        font-family: serif;
-        font-weight: bold;
-        margin: 16px 0;
-        line-height: 1.1;
-      }
-      .content .description {
-        font-size: 12px;
-        color: rgb(1, 5, 24);
-        font-family: sans-serif;
-        line-height: 1.5;
-      }
-      .content .price {
-        font-size: 36px;
-        color: rgb(5, 7, 61);
-        font-family: serif;
-        font-weight: bold;
-        margin: 16px 0;
-        line-height: 1.1;
-      } 
-      .content .discount {
-        font-size: 36px;
-        color: rgb(5, 7, 61);
-        font-family: serif;
-        font-weight: bold;
-        margin: 16px 0;
-        line-height: 1.1;
-      }
-      .content .real_price {
-        font-size: 24px;
-        color: rgba(20, 25, 81, 0.637);
-        font-family: serif;
-        font-weight: bold;
-        margin: 16px 0;
-        line-height: 1.1;
-      } 
-    </style>
-  </head>
-  <body style="background-color: rgba(3, 113, 172, 0.708);">
-    <div class="card"> 
-      <div class="image">
-      <img class="vergil" src="M1Muy23xZPg.jpg"/>
-      <div class="demotivation">VERGIL</div>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="https://code.iconify.design/iconify-icon/1.0.7/iconify-icon.min.js"></script>
+  <script src="https://cdn.tailwindcss.com"></script>
+  <title>Mega Motivation</title>
```

Comment:
Измените название на
```suggestion
  <title>Tailwind Card</title>
```